test/src/client/entity-editor/validators/helpers.js

```diff
@@ -150,7 +150,7 @@ export function testValidateEndDateFunc(
 			const result = INVALID_DATE_PAIR.reduce((res, datePair) =>
 				res || endDateValidationfunc(datePair.first, datePair.second),
 			false);
-			expect(result).to.be.false;
+			expect(result.isValid).to.be.false;
```

Comment:
Here and everywhere you use `.reduce` on a valid/invalid date array, shouldn't `.isValid` be checked in the reduce callback rather than on the result?
```
const result = INVALID_DATE_PAIR.reduce((res, datePair) =>
	res || endDateValidationfunc(datePair.first, datePair.second).isValid,
false);
expect(result).to.be.false;
```